Serializer/JsonLDNormalizer.php

```diff
@@ -165,7 +166,14 @@ protected function normalizeItem($data, $format, array $context)
         }
 
         if ($data instanceof \stdClass) {
-            return is_array($result) ? $result : [$result];
+            $result = is_array($result) ? $result : [$result];
```

Comment:
can't this bit live in MapNormalizser if it implemented serializer?

Reply:
This is specific to normalizing Objects rather than Maps so I think it should remain here and when we come to seperate the normalization/denormalization of types into seperate normalizers it should be in a normalizer called something like `JsonLDObjectNormalizer`